### DISCUSSION TITLE
How to set different api in different swagger page? e.g. the fronted api and backend api in the different page,  details shows in below image.

### DISCUSSION THREAD

**Discussion by zhiyixu** [fastapi/fastapi#13108] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
no code here, it's about swagger ui
```

### Description

the fronted api and backend api in the different page,  details shows in below image.

### Operating System

Windows

### FastAPI Version

0.115.6

### Pydantic Version

2.10.3

### Python Version

3.9

### Additional Context

![image](https://github.com/user-attachments/assets/b8c81b0f-b59e-4880-aca9-9f155d62f482)

**Comment by YuriiMotov** [comment, marked as answer]
You can [mount](https://fastapi.tiangolo.com/advanced/sub-applications/) frontend as a sub-app. This way docs will be separated and you will be able to access frontend docs via '/front/docs':
```
from fastapi import FastAPI

app = FastAPI(title="Back-end app")

front_app = FastAPI(title="Front-end app")
app.mount("/front", front_app)

```

**Reply by zhiyixu** [reply]
thanks for your help
